packages/web-components/fast-components/src/tabs/tabs.styles.ts

```diff
@@ -38,36 +38,26 @@ export const tabsStyles: (
 
         .tablist {
             display: grid;
-            grid-template-rows: auto auto;
+            grid-template-rows: calc(${heightNumber} * 1px); auto;
             grid-template-columns: auto;
+            grid-column-gap: 8px;
```

Comment:
Should this be something based off of density or designUnit?

Reply:
updated